dot/sync/chain_sync_integeration_test.go

```diff
@@ -41,8 +44,8 @@ func newTestChainSync(t *testing.T) (*chainSync, *blockQueue) {
 	bs.On("HasHeader", mock.AnythingOfType("common.Hash")).Return(true, nil)
 
 	net := new(syncmocks.Network)
-	net.On("DoBlockRequest", mock.AnythingOfType("peer.ID"),
-		mock.AnythingOfType("*network.BlockRequestMessage")).Return(nil, nil)
+	net.On("DoBlockRequest", mock.AnythingOfType("peer.ID"), mock.AnythingOfType("*network.BlockRequestMessage")).
+		Return(nil, nil)
```

Comment:
There are a lot of deltas due to line breaking up by your IDE, maybe could you disable this feature? This adds load for reviewers.
We already have the `lll` linter to ensure line don't go above 120chars.
And also reset back those changes for git blame purposes 😄